src/main/kotlin/no/nav/familie/ba/sak/kjerne/vilkårsvurdering/preutfylling/PreutfyllBosattIRiketService.kt

```diff
@@ -61,15 +61,28 @@ class PreutfyllBosattIRiketService(
 
         val erBosattOgHarNordiskStatsborgerskapTidslinje =
             erNordiskStatsborgerTidslinje.kombinerMed(erBosattINorgeTidslinje) { erNordisk, erBosatt ->
-                erNordisk == true && erBosatt == true
+                val nordiskOgBosatt = erNordisk == true && erBosatt == true
+                if (nordiskOgBosatt) {
+                    Delvilkår.OppfyltDelvilkår("- Norsk/nordisk statsborgerskap")
+                } else {
+                    Delvilkår.IkkeOppfyltDelvilkår
+                }
             }
 
         val erØvrigeKravForBosattIRiketOppfyltTidslinje = lagErØvrigeKravForBosattIRiketOppfyltTidslinje(erBosattINorgeTidslinje, personResultat)
 
         val erBosattIRiketTidslinje =
             erØvrigeKravForBosattIRiketOppfyltTidslinje
                 .kombinerMed(erBosattOgHarNordiskStatsborgerskapTidslinje) { erØvrigeKravOppfylt, erNordiskOgBosatt ->
-                    erØvrigeKravOppfylt == true || erNordiskOgBosatt == true
+                    val oppfylt = erØvrigeKravOppfylt?.erOppfylt == true || erNordiskOgBosatt?.erOppfylt == true
+
+                    val kommentar = listOf(erNordiskOgBosatt, erØvrigeKravOppfylt).mapNotNull { it?.begrunnelse?.takeIf { kommentar -> kommentar.isNotBlank() } }.joinToString("\n")
```

Comment:
Hvis `begrunnelse` er nullable kan denne fortkortes litt
```suggestion
                    val kommentar = listOfNotNull(erNordiskOgBosatt?.begrunnelse, erØvrigeKravOppfylt?.begrunnelse).joinToString("\n")
```